conf/iridanext.config

```diff
@@ -0,0 +1,21 @@
+iridanext {
+    enabled = true
+    output {
+        path = "${params.outdir}/iridanext.output.json.gz"
+        overwrite = true
+        files {
+            idkey = "irida_id"
+            global = [
+                "**/serotype/TypingQC_report.csv"
```

Comment:
In other pipelines we've also been including the versions YAML file `"**/pipeline_info/software_versions.yml"`. 

For example: https://github.com/phac-nml/gasclustering/blob/e61f072aaffe0de02cfce5b6e6a7c07758c01ced/conf/iridanext.config#L22